src/collector/database/models.py

```diff
@@ -18,18 +18,30 @@
 #   along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import logging
-from peewee import (Model, SqliteDatabase, PrimaryKeyField, IntegerField, FloatField,
+import os
+from playhouse.pool import PooledMySQLDatabase
+from peewee import (Model, PrimaryKeyField, IntegerField, BigIntegerField, DoubleField,
                     DateField, CharField)
-from src import DB_FILE_PATH
+import sys
+from time import sleep
 
 logger = logging.getLogger(__name__)
 
 
-class BaseModel(Model):
-    database = SqliteDatabase(DB_FILE_PATH)
+db = PooledMySQLDatabase(
```

Comment:
I'd recommend moving the db to a separate module, exposing it via a getter function, and then using that getter in BaseModel